org-gtd-backward-compatibility.el

```diff
@@ -23,10 +23,29 @@
 ;; Functions that don't exist in older vanilla emacsen
 ;;
 ;;; Code:
+
+;;;; Requirements
+
 (require 'subr-x)
 
+;;;; Functions
+
+;;;;; Private
+
+(defun org-gtd--ensure-list (object)
+  "Return OBJECT as a list.
+If OBJECT is already a list, return OBJECT itself.  If it's
+not a list, return a one-element list containing OBJECT."
+  (if (listp object)
+      object
+    (list object)))
+
+;; this was added in emacs 28.1
+(unless (fboundp 'ensure-list)
```

Comment:
This is not private logic; this is actual logic executed when the file is loaded. How does this fit? Does this need a new comment?

Reply:
Yeah, welcome to supporting multiple versions of emacs.